lib/content-services/src/lib/auth-loader/content-auth-loader.service.spec.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { AuthenticationService, BasicAlfrescoAuthService } from '@alfresco/adf-core';
+import { ContentAuthLoaderService } from './content-auth-loader.service';
+import { Subject } from 'rxjs';
+
+fdescribe('ContentAuthLoaderService', () => {
+    let service: ContentAuthLoaderService;
+    let authService: AuthenticationService;
+    let basicAlfrescoAuthService: BasicAlfrescoAuthService;
+    let onLoginSubject: Subject<void>;
+
+    beforeEach(() => {
+        onLoginSubject = new Subject<void>();
+        TestBed.configureTestingModule({
+            providers: [
+                ContentAuthLoaderService,
+                {
+                    provide: AuthenticationService,
+                    useValue: {
+                        onLogin: onLoginSubject.asObservable(),
+                        isOauth: () => false,
+                        isALLProvider: () => false,
+                        isECMProvider: () => false,
```

Comment:
I'm getting many unexpected comma eslint errors, please correct: 
<img width="1707" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/18d5c37c-75b1-4be1-981a-cbd727df8173">

Reply:
thanks for noticing, had my settings in webstorm restarted and it disable code refactoring